aws/table_aws_rolesanywhere_profile.go

```diff
@@ -0,0 +1,178 @@
+		List: &plugin.ListConfig{
+			Hydrate: listProfiles,
+			Tags:    map[string]string{"service": "rolesanywhere", "action": "ListProfiles"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(rolesanywherev1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The ID of the profile.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProfileId"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) specifying the profile.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ProfileArn"),
+			},
+			{
+				Name:        "name",
+				Description: "The name of the profile.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "accept_role_session_name",
+				Description: "Accept custom role session names.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The date and time when the profile was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "created_at",
+				Description: "The date and time when the profile was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "created_by",
+				Description: "The account that created the profile.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "duration",
+				Description: "The profile credential session duration in seconds.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("DurationSeconds"),
+			},
+			{
+				Name:        "enabled",
+				Description: "If the profile is enabled or not.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "attribute_mappings",
+				Description: "List of attribute mappings for certificate fields to session tags.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "managed_policy_arns",
+				Description: "List of managed IAM boundary policy ARNs attached to the profile.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "role_arns",
+				Description: "List of IAM role ARNs the profile can assume.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "require_instance_properties",
+				Description: "If instance properties are required for the session creation.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "session_policy",
+				Description: "Session policy applied to created sessions.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("SessionPolicy").Transform(transform.UnmarshalYAML),
+			},
+			{
+				Name:        "session_policy_std",
+				Description: "Contains the session policy in a canonical form for easier searching.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("SessionPolicy").Transform(unescape).Transform(policyToCanonical),
+			},
```

Comment:
Please include the steampipe standard column here. For reference: [aws_acm_certificate](https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_acm_certificate.go#L214-L232)